README.md

```diff
@@ -57,6 +57,26 @@ To stop services:
 sm2 --stop PILLAR2_ALL
 ```
 
+## Generation, Validation and Publishing
+To ensure API documentation alignment with code, we generate the OpenAPI specification (OAS) directly from the route definitions using an SBT task, reducing risk and manual effort.
+
+### Generation
+To generate the YAML OAS, run:
+```shell
+sbt routesToYamlOas
+```
+The generated OAS will include definitions based on the application's routes. The output folder is 'target/swagger' by default.
+
+### Validation
+To run basic validation on the generated OAS, run: 
+```shell
+sbt validateOas
```

Comment:
Does this validateOas task validate based on the spec found in target/swagger or on the one we manually need to move into resources/public/api/conf/1.0 ?
If the latter, it might be worth adding a comment regarding this.

Reply:
It validates the spec generated in `sbt routesToYamlOas`. I updated the PR to specify.